diff --git a/drivers/lora/CMakeLists.txt b/drivers/lora/CMakeLists.txt
index cda2f6dbaf3..18d04ec099a 100644
--- a/drivers/lora/CMakeLists.txt
+++ b/drivers/lora/CMakeLists.txt
@@ -5,5 +5,6 @@ zephyr_sources_ifdef(CONFIG_LORA_SHELL shell.c)
 
 add_subdirectory_ifdef(CONFIG_LORA_LR11XX lr11xx)
+add_subdirectory_ifdef(CONFIG_LORA_LR20XX lr20xx)
 
 # zephyr-keep-sorted-start
 add_subdirectory_ifdef(CONFIG_LORA_MODULE_BACKEND_LORAMAC_NODE loramac-node)
diff --git a/drivers/lora/Kconfig b/drivers/lora/Kconfig
index 0ebee4246f3..f8513b37044 100644
--- a/drivers/lora/Kconfig
+++ b/drivers/lora/Kconfig
@@ -62,5 +62,6 @@ config LORA_INIT_PRIORITY
 rsource "Kconfig.sx12xx"
 rsource "Kconfig.rylrxxx"
 rsource "lr11xx/Kconfig"
+rsource "lr20xx/Kconfig"
 rsource "lora-basics-modem/Kconfig"
 rsource "native/Kconfig"
